src/graph/ExecutionContext.h

```diff
@@ -40,6 +41,7 @@ class ExecutionContext final : public cpp::NonCopyable, public cpp::NonMovable {
         storageClient_ = storage;
         metaClient_ = metaClient;
         variableHolder_ = std::make_unique<VariableHolder>();
+        charsetInfo_ = CharsetInfo::instance();
```

Comment:
If you only use this instance here,  maybe we don't need the singleton.

Reply:
In pr  #1782，for expression compare need CharsetInfo,  so storage also need one instance .
in  charset display,   console need one instance

Reply:
I will move this singleton instance into graphd.
In In pr #1782  I has added  this singleton instance to storaged and CmdProcessor.
so Graphd、storaged、 CmdProcessor  have singleton instance.